.github/workflows/oclif-release.yml

```diff
@@ -48,17 +48,25 @@ jobs:
         id: sha256
         if: ${{matrix.publish == 'homebrew'}}
         run: |
-           npx oclif-dev publish
-           brew install coreutils
-           echo "::set-output name=sha256::$(sha256sum dist/${{ matrix.asset_name }}/${{ matrix.asset_name }}.tar.gz | awk '{print $1}')"
+          npx oclif-dev publish
+          brew install coreutils
+          if [ "${{github.event.inputs.formula}}" == "twiliorc" ]; then
+            echo "::set-output name=sha256::$(sha256sum dist/channels/rc/${{ matrix.asset_name }}/${{ matrix.asset_name }}.tar.gz | awk '{print $1}')"
+          else
+            echo "::set-output name=sha256::$(sha256sum dist/${{ matrix.asset_name }}/${{ matrix.asset_name }}.tar.gz | awk '{print $1}')"
+          fi
   home-brew-release:
     name: Trigger homebrew release workflow
     runs-on: ubuntu-latest
     needs: [oclif-release]
     steps:
       - name: Checkout cli repo
         uses: actions/checkout@v2
-      - name: Invoke HomeBrew workflow
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Invoke Scoop workflow
```

Comment:
update the name, `Invoke Scoop workflow` seems like a typo in `home-brew-release:`

Reply:
Thanks for pointing. When I am copying from my forked repo, copied to wrong place. Updated now.